hooks/publish.py

```diff
@@ -619,7 +619,7 @@ def publish_files(self, task_settings, item, publish_path):
         else:
             work_files = [path]
 
-        return publisher.util.copy_files(work_files, publish_path, is_sequence)
+        return publisher.util.copy_files(work_files, publish_path, seal_files=False, is_sequence=is_sequence)
```

Comment:
Add this:

```python
# Determine if we should seal the copied files or not
seal_files = item.properties.get("seal_files", False)

return publisher.util.copy_files(work_files, publish_path, seal_files, is_sequence)
```

That way we can enable this later on (on a per item basis) without have to modify core code.